src/main/java/com/academia/library/exception/BookNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package com.academia.library.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
I guess this is redundant here because you can set is in handler. Please do that

Reply:
Ok, thank